module1/build tools/ustils/src/main/java/com/epam/task1/StringUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.task1;
+
+import org.apache.commons.lang3.math.NumberUtils;
+
+public class StringUtils {
+
+    public static void main(String[] args) {
+
+    }
+
+    public static boolean isPositiveNumber(String str) {
+        boolean result;
+        if(NumberUtils.isParsable(str)) {
+            Long fromString = NumberUtils.createLong(str);
+            result = fromString > 0;
+        }else{
+            result = false;
+        }
+        return result;
```

Comment:
This 8 rows of code can be simplified to one.

Reply:
Simplified method to one line.

Reply:
@pilinoga can not see ustils module inside 'build tools' directory anymore. Only multi-project left

Reply:
@senator2202 returned utils module to remote repository.